Formula/circleci.rb

```diff
@@ -0,0 +1,47 @@
+class Circleci < Formula
+  desc "Tool which enables to reproduce CircleCI build locally"
+  homepage "https://circleci.com/docs/2.0/local-jobs/"
+  url "https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci"
```

Comment:
This doesn't seem like a URL that is going to remain stable over time? You need a versioned URL, not one that will always point to the latest.

Reply:
This is the one given by CircleCI itself https://circleci.com/docs/2.0/local-jobs/ no version specific URL seems to be available.

Reply:
You'll need to ask them for a versioned URL.

Reply:
I've submitted a (PR that would create versioned URLs([https://github.com/circleci/local-cli/issues/7] for use by a formula, and have it working against a (tap in my own repo)[https://github.com/eddiewebb/homebrew-circleci]

If that PR gets merged I will submit a new PR against core. the tap formula is passing all audits except repo age (< 30 days)